src/objects/sceneManager.js

```diff
@@ -0,0 +1,68 @@
+import { MAIN_MENU, CHEAT_SHEET, CHALLENGE_MODE, GAME_OVER, INSTRUCTIONS, PRACTICE_MODE, SETTINGS } from "../constants/sceneConstants.js";
+import MainMenu from "../scenes/mainMenu.js";
+import CheatSheet from "../scenes/cheatSheet.js";
+import GameOver from "../scenes/gameOver.js";
+import Instructions from "../scenes/instructions.js";
+import PracticeMode from "../scenes/practiceMode.js";
+import Settings from "../scenes/settings.js";
+import ChallengeMode from "../scenes/challengeMode.js";
+
+/**
+ * We'll use this class to help us manage our scenes;
+ * It will expose a simple API, and hide its inner workings
+ * to any callers.
+ */
+class SceneManager { 
+
+    // Putting the '#' in front of our property here makes it a private property, accessible only to this class.
```

Comment:
You can read more about private class fields here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields